keygen/openssl.conf

```diff
@@ -1,4 +1,41 @@
 [req]
 distinguished_name = req_distinguished_name
+x509_extensions = v3_ca	# The extentions to add to the self signed cert
 
 [req_distinguished_name]
+
+[ v3_ca ]
```

Comment:
I would prefer that we don't use spaces inside section names. It could be just `[v3_ca]`